_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -31,61 +31,76 @@ separate documents will cover these advanced configurations.
 Setup consists of preparing the mesh for expansion and installing and configuring each VM.
 
 An example script to help with Kubernetes setup is available in
-[install/tools/setupMeshEx.sh](https://raw.githubusercontent.com/istio/istio/master/install/tools/setupMeshEx.sh).
+[install/tools/setupMeshEx.sh](https://raw.githubusercontent.com/istio/istio/master/install/tools/setupMeshEx.sh). Please check the script content and environment variables supported (like GCP_OPTS)
 
 An example script to help configure a machine is available in [install/tools/setupIstioVM.sh](https://raw.githubusercontent.com/istio/istio/master/install/tools/setupIstioVM.sh).
 You should customize it based on your provisioning tools and DNS requirements.
 
 ### Preparing the Kubernetes cluster for expansion
 
-* Setup internal load balancers for Kube DNS, Pilot, Mixer and CA. This step is specific to
-each cluster, you may need to add annotations.
+* Setup Internal Load Balancers (ILBs) for Kube DNS, Pilot, Mixer and CA. This step is specific to
+each cloud provider, you may need to edit annotations.
+
+> The yaml part of the 0.2.7 distribution has a typo with the wrong
+namespace for the DNS ILB, use
+[this one](https://raw.githubusercontent.com/istio/istio/master/install/kubernetes/mesh-expansion.yaml),
+likewise for the setupMeshEx.sh, please use the latest above or from cloning [GitHub.com/istio/istio](https://github.com/istio/istio/)
```

Comment:
The .yaml file of the 0.2.7 distribution has an incorrect namespace for the DNS ILB. Use [this one] (..) instead. The `setupMeshEx.sh` also has a typo. Use the latest file from the link above or from cloning ....